Bottom: d412c2b91ec3779887b31035e1ea6fbf03021fef
Top:    7e622bd11f74e999615676332dcb670c6c0a38f1
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-08 11:41:00 -0500

Refresh of ldap-add-phpldapadmin

---
diff --git a/roles/docker-openldap/tasks/main.yml b/roles/docker-openldap/tasks/main.yml
index 2d6a09d9..79921674 100644
--- a/roles/docker-openldap/tasks/main.yml
+++ b/roles/docker-openldap/tasks/main.yml
@@ -137,6 +137,7 @@
     name: '{{ phpldapadmin_image_name }}'
     tag: '{{ docker_image_tag }}'
     path: '{{ docker_projects_dir }}/docker-phpldapadmin'
+    force: "{{ docker_image_force_build }}"
 
 - name: start container (phpldapadmin)
   docker_container:
